src/flash-source-buffer.js

```diff
@@ -412,4 +412,3 @@ export default class FlashSourceBuffer extends videojs.EventTarget {
     return tags;
   }
 }
-
```

Comment:
Did you remove the trailing newline? It's hard to tell if that is what was happening here.